pkg/asset/cluster/tfvars.go

```diff
@@ -890,28 +897,29 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		osImage := strings.SplitN(string(*rhcosImage), "/", 2)
 		data, err = powervstfvars.TFVars(
 			powervstfvars.TFVarsSources{
-				MasterConfigs:         masterConfigs,
-				Region:                installConfig.Config.Platform.PowerVS.Region,
-				Zone:                  installConfig.Config.Platform.PowerVS.Zone,
-				APIKey:                APIKey,
-				SSHKey:                installConfig.Config.SSHKey,
-				PowerVSResourceGroup:  installConfig.Config.PowerVS.PowerVSResourceGroup,
-				ImageBucketName:       osImage[0],
-				ImageBucketFileName:   osImage[1],
-				NetworkName:           installConfig.Config.PowerVS.PVSNetworkName,
-				VPCRegion:             vpcRegion,
-				VPCZone:               vpcZone,
-				VPCName:               vpcName,
-				VPCSubnetName:         vpcSubnet,
-				VPCPermitted:          vpcPermitted,
-				VPCGatewayName:        vpcGatewayName,
-				VPCGatewayAttached:    vpcGatewayAttached,
-				CISInstanceCRN:        cisCRN,
-				DNSInstanceCRN:        dnsCRN,
-				PublishStrategy:       installConfig.Config.Publish,
-				EnableSNAT:            len(installConfig.Config.DeprecatedImageContentSources) == 0 && len(installConfig.Config.ImageDigestSources) == 0,
-				TransitGatewayEnabled: transitGatewayEnabled,
-				ServiceInstanceCRN:    serviceInstanceCRN,
+				MasterConfigs:          masterConfigs,
+				Region:                 installConfig.Config.Platform.PowerVS.Region,
+				Zone:                   installConfig.Config.Platform.PowerVS.Zone,
+				APIKey:                 APIKey,
+				SSHKey:                 installConfig.Config.SSHKey,
+				PowerVSResourceGroup:   installConfig.Config.PowerVS.PowerVSResourceGroup,
+				ImageBucketName:        osImage[0],
+				ImageBucketFileName:    osImage[1],
+				NetworkName:            installConfig.Config.PowerVS.PVSNetworkName,
+				VPCRegion:              vpcRegion,
+				VPCZone:                vpcZone,
+				VPCName:                vpcName,
+				VPCSubnetName:          vpcSubnet,
+				VPCPermitted:           vpcPermitted,
+				VPCGatewayName:         vpcGatewayName,
+				VPCGatewayAttached:     vpcGatewayAttached,
+				CISInstanceCRN:         cisCRN,
+				DNSInstanceCRN:         dnsCRN,
+				PublishStrategy:        installConfig.Config.Publish,
+				EnableSNAT:             len(installConfig.Config.DeprecatedImageContentSources) == 0 && len(installConfig.Config.ImageDigestSources) == 0,
+				TransitGatewayEnabled:  transitGatewayEnabled,
```

Comment:
Is this still used if you're passing the `AttachedTransitGateway`?

Reply:
Info about "enabled" or not still controls whether it wants to create a Cloud Connection. So, until we retire that part of the logic completely, I suspect we want to keep computing and passing it in to TF.

Reply:
Got it! Yep we'll leave that for now.